### PR TITLE
Add new read or die cards.

### PR DESCRIPTION
**Pull request by zzorba**
New parallel Daisy Walker + the 4 new encounter cards. I set up daisy to
have the replacement cards, but these aren't quite replacements (you can
have either or, not both).

### CODE REVIEW

**Comment by zzorba** [comment]
@drawntotheflame if you could look over these, I think we're still figuring out exactly how to support Daisy but I typed up the encounter cards in the mean time.

File: pack/parallel/rod.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+[
+	{
+		"back_flavor": "As a respected librarian at Miskatonic University, Daisy had always felt that books were the most important thing in her life. She explored in fiction what she abhorred in life: horror, violence, fear. Then, she stumbled across the John Dee translation of the Necronomicon. It was blasphemous, unholy, and too awful to be read. But given her studies in obscure and occult subjects, Daisy knew there was more truth than fiction within the book's pages. She began to wonder what other secrets the restricted collection of the Orne Library held...",
```
Comment (drawntotheflame) [inline pack/parallel/rod.json:3]: Should we underline 'Necronomicon' to make clear it's not italicised?

File: pack/parallel/rod.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		"cost": 2,
+		"deck_limit": 1,
+		"faction_code": "neutral",
+		"illustrator": "Dani Hartel",
+		"is_unique": true,
+		"name": "Daisy's Tote Bag",
+		"pack_code": "rod",
+		"position": 2,
+		"quantity": 1,
+		"restrictions": "investigator:90001",
```
Comment (drawntotheflame) [inline pack/parallel/rod.json:44]: Does this restriction need to include 01002 (original Daisy)?

File: pack/parallel/rod.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+		"skill_intellect": 1,
+		"skill_wild": 1,
```
Comment (drawntotheflame) [inline pack/parallel/rod.json:46]: wild is 2

File: pack/parallel/rod.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+		"skill_willpower": 2,
```
Comment (drawntotheflame) [inline pack/parallel/rod.json:47]: willpower is 1

File: pack/parallel/rod.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+	{
+		"code": "90003",
+		"deck_limit": 1,
+		"faction_code": "neutral",
+		"illustrator": "Jesse Mead",
+		"name": "The Necronomicon",
+		"pack_code": "rod",
+		"position": 3,
+		"quantity": 1,
+		"restrictions": "investigator:90001",
```
Comment (drawntotheflame) [inline pack/parallel/rod.json:61]: Same as above - should think include 01002?

File: pack/parallel/rod.json [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+		"skill_intellect": 1,
+		"skill_wild": 1,
+		"skill_willpower": 2,
+		"text": "Daisy Walker deck only. Advanced.\nYou have 2 additional hand slots, which can only be used to hold [[Tome]] assets.\n[reaction] When you play a [[Tome]] asset during your turn, exhaust Daisy's Tote Bag: That asset gains fast.",
+		"traits": "Item.",
+		"type_code": "asset"
+	},
+	{
+		"code": "90003",
+		"deck_limit": 1,
```
Comment (drawntotheflame) [inline pack/parallel/rod.json:54]: cost is 'null'

File: pack/parallel/rod_encounter.json [hunk 7]
```diff
@@ -0,0 +1,77 @@
+[
+    {
+        "back_text": "<b>Hard / Expert</b>\n[skull]: -X. X is 1 more than the number of [[Tome]] assets Daisy Walker controls.\n[cultist]: Reveal another token. If you fail, discard the top 3 cards of your deck.\n[tablet]: -3. If this is an attack or evasion attempt against Namer of the Dead and you do not succeed by at least 3, it attacks you.\n[elder_thing]: -5. If you fail, Daisy Walker takes 1 horror.",
```
Comment (drawntotheflame) [inline pack/parallel/rod_encounter.json:3]: No need for bold tags around 'hard / expert' (we changed style halfway through Dunwich - I need to remove the bold tags from the first few scenario cards)

File: pack/parallel/rod_encounter.json [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+        "code": "90004",
+        "double_sided": true,
+        "encounter_code": "read_or_die",
+        "encounter_position": 1,
+        "faction_code": "mythos",
+        "name": "Read or Die",
+        "pack_code": "rod",
+        "position": 4,
+        "quantity": 1,
+        "text": "<b>Easy / Standard</b>\n[skull]: -X. X is the number of [[Tome]] assets Daisy Walker controls.\n[cultist]: Reveal another token. If you fail, discard the top 2 cards of your deck.\n[tablet]: -2. If this is an attack or evasion attempt against Namer of the Dead and you do not succeed by at least 2, it attacks you.\n[elder_thing]: -3. If you fail, Daisy Walker takes 1 horror.",
```
Comment (drawntotheflame) [inline pack/parallel/rod_encounter.json:13]: Same with bold tags.

File: pack/parallel/rod_encounter.json [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+        "type_code": "scenario"
+    },
+    {
+        "back_flavor": "A tremendous whirlwind and a thunderous voice erupts from the Necronomicon, sending a tornado of pages into the sky. A sinister shadow looms over the entire campus, then escapes through the ensuing tempest. When it is done, the grimoire clatters to the floor, devoid of energy. Whatever presence it held before is now gone. It feels somehow empty in your hands.\n",
```
Comment (drawntotheflame) [inline pack/parallel/rod_encounter.json:17]: There's a spare '\n' tag at the end of this line.

File: pack/parallel/rod_encounter.json [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+        "name": "Mortal Inquiry",
+        "pack_code": "rod",
+        "position": 5,
+        "quantity": 1,
+        "stage": 1,
+        "text": "Dr. Henry Armitage does not take up an ally slot and gains: \"You have 2 additional hand slots, which can only be used to hold [[Tome]] assets.\"\n\"Jazz\" Mulligan gains: \"[action]: Move to a [[Miskatonic]] location up to 3 connections away.\"",
+        "type_code": "agenda"
+    },
+    {
+        "back_flavor": "You stand in the restricted section of the Orne Library, surrounded by volumes of ancient, heretical texts. The incantations you have found may prove to be enough to disep the entity—or perhaps bind it back within the Necronomicon from whence it came. Your voice rises to a crescendo as you repeat the words. Shadows crawl across the room. The walls and ceiling bend. Pages flutter in the temptuous gale. Every fiber of your hair is whisked into the air. All is swept away in the ensuing chaos, like a wet paintbrush kissing the canvas of reality.",
```
Comment (drawntotheflame) [inline pack/parallel/rod_encounter.json:37]: typo: 'disep' to 'dispel'

Comment (drawntotheflame) [reply pack/parallel/rod_encounter.json:37]: Also, please add spaces around the dashes: 'entity - or' rather than 'entity-or'

Comment (drawntotheflame) [reply pack/parallel/rod_encounter.json:37]: Typo: 'temptuous' to 'tempestuous'

File: pack/parallel/rod_encounter.json [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+        "back_name": "Page-turner",
+        "back_text": "<b>(→R1)</b>",
+        "clues": null,
+        "code": "90006",
+        "double_sided": true,
+        "encounter_code": "read_or_die",
+        "encounter_position": 3,
+        "faction_code": "mythos",
+        "flavor": "The way to seal this being away must be recorded in one of the main tomes around campus...",
```
Comment (drawntotheflame) [inline pack/parallel/rod_encounter.json:46]: Please change 'main' to 'many'

**Comment by drawntotheflame** [comment]
@Kamalisk @zzorba here's the big question: does Parallel get its own section, like TCU or any other cycle, or should we nest it under 'Promotional', so we have 'books', 'promo' and 'parallel' there?

File: pack/parallel/rod.json (hunk 2 above)
Comment (zzorba) [reply pack/parallel/rod.json:44]: So the existing promos only reference the 'original' card. @Kamalisk what do you think we should do for these Advanced ones, same thing?

**Comment by Kamalisk** [comment]
For now they should point to the original cards for most purposes.

**Pull request merged**